jekyll/_cci2/basics.md

```diff
@@ -31,6 +31,7 @@ Document | Description
 <a href="{{ site.baseurl }}/2.0/env-vars/">Using Environment Variables</a> | How to use environment variables in the CircleCI app and in the `config.yml` file.
 <a href="{{ site.baseurl }}/2.0/contexts/">Using Contexts</a> | How to use Contexts to set global environment variables.
 <a href="{{ site.baseurl }}/2.0/notifications/">Enabling Notifications</a> | How to set or modify Slack, chat, and email notifications in the CircleCI app.
+[Embedding Build Status Badges]({{ site.baseurl }}/2.0/status-badges) | How to display the status of your builds outside of CircleCI.
```

Comment:
change 'outside of CircleCI' to 'on a web page or document' to be more clear and specific